src/Resources/app/administration/src/module/findologic-module/page/findologic-page/index.js

```diff
@@ -342,12 +343,16 @@ Component.register('findologic-page', {
                 this.selectedSalesChannelId = salesChannelId;
                 this.onSelectedLanguage(selectedChannel.languageId);
                 selectedChannel.languages.forEach((language) => {
-                    this.language.push({
-                        name: language.name,
-                        label: language.name,
-                        value: language.id
-                    });
-
+                    let domain = selectedChannel.domains.find(item => item.languageId === language.id);
```

Comment:
Always prefer `const` over `let`

```suggestion
                    const domain = selectedChannel.domains.find(item => item.languageId === language.id);
```